### ISSUE TITLE
Error with strictNullChecks enabled

### ISSUE THREAD

**Issue by codeandcats** [typestyle/typestyle#16]
If consuming project has strictNullChecks enabled it will error on the typestyle typings:

In: `typestyle/src/css.d.ts`
```
(1440,3): error TS2411: Property ''&:hover'' of type 'NestedCSSProperties | undefined' is not assignable to string index type 'string | number | Nested
CSSProperties | ReactText[]'.
```

**Comment by notoriousb1t** [comment]
I added strictNullChecks in 4990254f1a50e4aeeeeafc2a80d5aaed967a0240 and made corrections to the css.d.ts. I will add an update here when it gets pushed out.

**Comment by basarat** [comment]
@notoriousb1t run `npm version patch` from the project folder whenever you have the time ;). Mention me if any help is required :rose:

**Comment by notoriousb1t** [comment]
This has been fixed in v0.4.2 (https://github.com/typestyle/typestyle/releases/tag/v0.4.2).  

Thank you for reporting this! 👍

**Comment by codeandcats** [comment]
Cheers for the quick fix guys
